test/spec/desktop/windowHandleSize.js

```diff
@@ -2,7 +2,19 @@ import conf from '../../conf/index'
 
 describe('windowHandleSize', () => {
     it('should change window size of current window if no handle is given', async function () {
-        await this.client.windowHandleSize({ width: 500, height: 500 })
```

Comment:
I am confused, why does this test has passed in the past? I just ran it on the current master branch and it works fine. So what's the actual problem then?